### PR TITLE
Fix migrate cycling views

### PR DESCRIPTION
**Pull request by ondrajodas**
Jira - https://keboola.atlassian.net/browse/CM-697

### CODE REVIEW

File: src/Migrate.php [hunk 1]
```diff
@@ -1631,26 +1631,37 @@ private function copyViews(string $database, string $databaseRole, array $viewsG
...
                     Helper::quoteIdentifier($view['database_name']),
-                    Helper::quoteIdentifier($view['schema_name']),
-                    Helper::quoteIdentifier($view['name']),
-                    $e->getMessage()
+                    Helper::quoteIdentifier($view['schema_name'])
                 ));
+
+                try {
+                    $this->destinationConnection->query($view['text']);
+                    unset($views[$viewKey]);
```
Comment (pivnicek) [inline src/Migrate.php:1653]: Clever!  So if it fails to create the view, then it remains in the list and the foreach continues until the end of the list.  Then it comes around (with +1 try) and tries to create whatever views are remaining that failed the first time.  Great stuff OJ.

**Review by pivnicek — DISMISSED** [review]
👍

**Comment by ondrajodas** [comment]
@pivnicek můžeš ještě jednou prosím?

**Review by pivnicek — APPROVED** [review]
jasny 👍

**Pull request merged**